src/ast_pattern.mli

```diff
@@ -215,8 +215,7 @@ val single_expr_payload : (expression, 'a, 'b) t -> (payload, 'a, 'b) t
 
 val no_label : (expression, 'a, 'b) t -> (Asttypes.arg_label * expression, 'a, 'b) t
 
-val attribute : (string, 'a, 'b) t -> (payload, 'b, 'c) t -> (attribute, 'a, 'c) t
-val extension : (string, 'a, 'b) t -> (payload, 'b, 'c) t -> (attribute, 'a, 'c) t
+val extension : (string, 'a, 'b) t -> (payload, 'b, 'c) t -> (extension, 'a, 'c) t
```

Comment:
That diff seems wrong: I think we still want a combinator to match attributes.

Reply:
Fixed.